### PR TITLE
dir: add page

### PR DESCRIPTION
**Pull request by amarjeetsinha**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Yes, dir command runs on linux too.

### CODE REVIEW

File: pages/linux/dir.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# dir
+
+> dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
+> Show the contents of the current directory; Works as "ls -C -b" in Linux.
+> More information: https://www.geeksforgeeks.org/dir-command-in-linux-with-examples/.
```
Comment (navarroaxel) [inline pages/linux/dir.md:5]:
```suggestion
> More information: <https://manned.org/dir>.
```

File: pages/linux/dir.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Example description:
+
+dir -l --author 
+Displays author of all the files. -l is required to display the contents in the form of a list.
```
Comment (navarroaxel) [inline pages/linux/dir.md:10]: I think is better to split this in 2 examples:
```suggestion
- List files using long list format:

`dir -l`

- List files including the author (`-l` is required):

`dir -l --author`
```

File: pages/linux/dir.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# dir
+
+> dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
+> Show the contents of the current directory; Works as "ls -C -b" in Linux.
+> More information: https://www.geeksforgeeks.org/dir-command-in-linux-with-examples/.
+
+- Example description:
```
Comment (mfrw) [inline pages/linux/dir.md:7]: Can we please follow the convention.
Please take a look at a few examples.

**Review by mfrw — CHANGES_REQUESTED** [review]

File: pages/linux/dir.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+dir -l --author 
+Displays author of all the files. -l is required to display the contents in the form of a list.
+
+- Example description:
+
+ dir -R 
+List subdirectories recursively.
```
Comment (navarroaxel) [inline pages/linux/dir.md:15]:
```suggestion
- List subdirectories recursively:

`dir --recursive`
```

File: pages/linux/dir.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# dir
+
+> dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
```
Comment (navarroaxel) [inline pages/linux/dir.md:3]:
```suggestion
> List directory contents using one line per file.
```

File: pages/linux/dir.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+> Show the contents of the current directory; Works as "ls -C -b" in Linux.
```
Comment (navarroaxel) [inline pages/linux/dir.md:4]:
```suggestion
> Works as `ls -C --escape`.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @amarjeetsinha, welcome and thanks for your submission. We have a specific format for the `tldr-pages` but the GitHub CI [is not working properly](https://twitter.com/githubstatus/status/1445384611606016008) at the moment and it's affecting our linter tool.

I left some comments!

**Comment by amarjeetsinha** [comment]
I've edited and followed all the conventions for this command to be listed in the file.
Kindly verify and merge.

File: pages/linux/dir.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+`dir -R`
+
+- Display the help options and exit:
+
+`dir --help`
+
+- Outputs version information and exit:
+
+`dir --version`
```
Comment (navarroaxel) [inline pages/linux/dir.md:44]: Sorry but the pages can have up to 8 examples. It's a strict rule.

Comment (navarroaxel) [inline pages/linux/dir.md:44]: Sorry but the pages can have up to 8 examples. It's a strict rule.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by amarjeetsinha** [comment]
Kindly review the changes @navarroaxel and @mfrw

File: pages/linux/dir.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# dir
+
+> Dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
+> More information: <https://manned.org/dir>.
+
+- Displays all the hidden files along with two files denoted by "." and "..":
```
Comment (navarroaxel) [inline pages/linux/dir.md:6]:
```suggestion
- List all files, including hidden files:
```

File: pages/linux/dir.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir -a`
+
+- Displays author of all the files. -l is required to display the contents in the form of a list:
```
Comment (navarroaxel) [inline pages/linux/dir.md:10]: I think you should add the `dir -l` example to explain that argument before `-author`
```suggestion
- List files including the author (`-l` is required):
```

File: pages/linux/dir.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir -l --author`
+
+- Ignore files describe by shell PATTERN while list the contents of a directory:
+
+`dir --hide=PATTERN`
+
+- List subdirectories recursively.:
```
Comment (navarroaxel) [inline pages/linux/dir.md:18]:
```suggestion
- List subdirectories recursively:
```

File: pages/linux/dir.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+# dir
+
+> Dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
+> More information: <https://manned.org/dir>.
+
+- Displays all the hidden files along with two files denoted by "." and "..":
+
+`dir -a`
```
Comment (navarroaxel) [inline pages/linux/dir.md:8]: We prefer long arguments if available
```suggestion
`dir --all`
```

File: pages/linux/dir.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir -l --author`
+
+- Ignore files describe by shell PATTERN while list the contents of a directory:
+
+`dir --hide=PATTERN`
+
+- List subdirectories recursively.:
+
+`dir -R`
```
Comment (navarroaxel) [inline pages/linux/dir.md:20]:
```suggestion
`dir --recursive`
```

File: pages/linux/dir.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display the help options and exit:
```
Comment (navarroaxel) [inline pages/linux/dir.md:22]: We have this as a standard description
```suggestion
- Display help:
```

File: pages/linux/dir.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+- Displays all the hidden files along with two files denoted by "." and "..":
+
+`dir -a`
+
+- Displays author of all the files. -l is required to display the contents in the form of a list:
+
+`dir -l --author`
+
+- Ignore files describe by shell PATTERN while list the contents of a directory:
```
Comment (navarroaxel) [inline pages/linux/dir.md:14]: I this this is more clear
```suggestion
- List files excluding those that match a specified pattern:
```

File: pages/linux/dir.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+`dir --hide=PATTERN`
```
Comment (navarroaxel) [inline pages/linux/dir.md:16]: We use `snake_case` for placeholders
```suggestion
`dir --hide={{pattern}}`
```

File: pages/linux/dir.md [hunk 16]
```diff
@@ -0,0 +1,24 @@
+# dir
+
+> Dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
```
Comment (navarroaxel) [inline pages/linux/dir.md:3]: Perhaps this better
```suggestion
> List directory contents using one line per file, special characters are represented by backslash escape sequences.
> Works as `ls -C --escape`.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by amarjeetsinha** [comment]
@navarroaxel and @mfrw can you merge this file, I'll update the file as soon as you merge it. Because I'm unable to update any changes. But I assure you that I'll update this file.
Screenshot (22)

**Comment by navarroaxel** [comment]
@amarjeetsinha Yeah Looks like your branch was deleted. Sorry, you should re-create the PR. Don't worry, let us know if you need any help.

**Comment by navarroaxel** [comment]
@amarjeetsinha I found a solution, I merged it in a temporal branch https://github.com/tldr-pages/tldr/tree/pages/dir

**Comment by amarjeetsinha** [comment]
Thanks @navarroaxel I'm updating the correction right now.

**Pull request merged**